test/test_cli.rb

```diff
@@ -49,7 +49,7 @@ def test_control_for_tcp
     cli = Puma::CLI.new ["-b", "tcp://127.0.0.1:0",
                          "--control-url", url,
                          "--control-token", "",
-                         "test/rackup/lobster.ru"], @log_writer, @events
+                         "test/rackup/hello.ru"], @log_writer, @events
```

Comment:
Why this change?

Reply:
I believe all the tests involve a control server, so the app isn't important?

Reply:
Just trying to understand the motivation behind making the change (to learn something)... has `Rack::Lobster` been removed?

Reply:
It's now part of rackup.  https://github.com/rack/rackup/blob/main/lib/rackup/lobster.rb

Sorry, when Rack v3.0.0 was released, I was in the middle of some Puma code, and wasn't too happy.

Maybe we should add a CI job that allows pre-releases and all dependencies are `>=`.  Give us better 'early warning'?

Reply:
Sure